pkg/api/v1alpha1/cloudstackdatacenterconfig_types.go

```diff
@@ -55,9 +55,12 @@ type CloudStackZone struct {
 	Network CloudStackResourceIdentifier `json:"network"`
 }
 
+// Important: Run "make" to regenerate code after modifying this file
+
 // CloudStackDatacenterConfigStatus defines the observed state of CloudStackDatacenterConfig
 type CloudStackDatacenterConfigStatus struct { // INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
+	// UUID of the CloudStack domain
+	DomainId string `json:"domainId"`
```

Comment:
why is this in status? Shouldn't it be in spec, in line with the existing "domain" attribute, or expanded to be a CloudStackResourceIdentifier? (line 30). At the very least, we should include a better description of the behavior for the customer for input domain